CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ Happily accepting pull requests; to edit the core library, modify files in `./sr
 
 On ARM based devices, the `gl` module may require some libraries to be re-installed:
 
-`sudo apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config` -- see https://github.com/stackgl/headless-gl#ubuntudebian for more.
+`sudo apt-get install --fix-missing -y build-essential xserver-xorg-dev libxext-dev libxi-dev libglu1-mesa-dev libglew-dev pkg-config` -- see https://github.com/stackgl/headless-gl#ubuntudebian for more.
```

Comment:
```suggestion
`sudo apt-get install -y build-essential xserver-xorg-dev libxext-dev libxi-dev libglu1-mesa-dev libglew-dev pkg-config` -- see https://github.com/stackgl/headless-gl#ubuntudebian for more.
```
I don't think the fix-missing is required.

Reply:
@HarshKhandeparkar thank you for helping. I got a pretty good idea of raising pull-request and working of Travis-ci. It was an awesome experience. Looking forward to more contribution in the future.